website/docs/cli/commands/validate.mdx

```diff
@@ -89,7 +89,7 @@ stream. The top-level JSON object will have the following properties:
 
 The nested objects in `diagnostics` have the following properties:
 
-- `severity` (string): A string keyword, currently either `"error"` or
+- `severity` (string): A string keyword, either `"error"` or
   `"warning"`, indicating the diagnostic severity.
```

Comment:
This one does seem like a place where we might have new severities in future. For example, my prototype in #29661 introduced a new diagnotic type "hint".

However, I suspect we'll do that only if the user opts in to it by activating some new feature, and so nothing built against a current Terraform workflow should encounter it and so hopefully this is okay.